client/blocks/login/index.jsx

```diff
@@ -338,6 +344,24 @@ class Login extends Component {
 			postHeader = <VisitSite siteSlug={ fromSite } />;
 		}
 
+		if ( isGutenboarding ) {
+			preHeader = (
+				<div class="login__form-gutenboarding-wordpress-logo">
+					<svg
```

Comment:
`Gridicon` is being used elsewhere in login already, which should have a WordPress logo in it. Can we re-use [that one](https://github.com/automattic/wp-calypso/blob/master/client/login/wp-login/index.jsx#L170) ( `<Gridicon icon="my-sites" size={ 18 } />` ), or is it better to inline the svg here? It's used in the footer if you go to crowdsignal.com and click login.

Reply:
That's what I was looking for! Why they didn't put "wordpress" in the name I will never know

Reply:
The logos aren't _quite_ the same. Even if I stretch the gridicon logo to make it a little bigger, they won't be quite identical.

![Apr-01-2020 10-19-25](https://user-images.githubusercontent.com/1500769/78076166-61ee8680-7402-11ea-9132-0213708d616e.gif)

I'm leaning towards using the grid icon anyway. Embedding the SVG means every user has to download the image, regardless of if they're in the gutenboarding flow or not. Hopefully the user will blink and miss it.

Reply:
Would this work?

```js
import { Icon } from '@wordpress/components';

<Icon icon="wordpress-alt" size={ 24 } />
```

I think it's available in the new icon package, too: https://github.com/WordPress/gutenberg/tree/master/packages/icons

Reply:
Even better! I didn't think that was going to work because I assumed there'd be some special icon related css that wouldn't be included in this bundle.

Reply:
Alas, it looks like using `<Icon>` is the way to go, but it breaks the node server.

The login page does some SSR, which means the `<Icon>` code is pulled in on the server side, which means a [priority-queue bug (which prevents it from running in node)](https://github.com/WordPress/gutenberg/pull/20486) is included on the server.

Looks like priority-queue needs to be updated in our package-lock before we can use `<Icon>`. So I'll leave it inline for now and update to `<Icon>` later.